flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          default: ipv4_prefix
+          x-enum:
+            ipv4_prefix:
+              x-field-uid: 1
+            four_byte_as_number:
+              x-field-uid: 2
+        ipv4_prefix:
+          $ref: '#/components/schemas/Flow.RSVP.PathExplicitRouteType1Ipv4Prefix'
+          x-field-uid: 2
+        four_byte_as_number:
+          $ref: '#/components/schemas/Flow.RSVP.PathExplicitRouteType1FourByteASNumber'
+          x-field-uid: 3
+    Flow.RSVP.PathExplicitRouteType1Ipv4Prefix:
+      description: >-
+        Class = EXPLICIT_ROUTE, Type1 ROUTE_RECORD C-Type = 1
+        Subobject: IPv4 Prefix, C-Type: 1
+      type: object
+      properties:
+        l_bit:
+          x-field-pattern:
+            description: >-
+              The L bit is an attribute of the subobject.The L bit is set if the subobject represents a loose hop in the explicit route.
+              If the bit is not set, the subobject represents a strict hop in the explicit route.
+            format: integer
+            length: 1
+            default: 0
+            features: [count]
+          x-field-uid: 1
+        length:
+          description: >-
+            The Length contains the total length of the subobject in bytes,including L ,Type and Length fields.  
+            The Length MUST be atleast 4, and MUST be a multiple of 4.
+          $ref: '#/components/schemas/Flow.RSVPExplicitRoute.Length'
+          x-field-uid: 2
+        ipv4_address:
+          x-field-pattern:
+            description: >-
+              This IPv4 address is treated as a prefix based on the prefix length value below. 
+              Bits beyond the prefix are ignored on receipt and SHOULD be set to zero on transmission.
+            format: ipv4
+            default: 0.0.0.0
+            features: [count]
+          x-field-uid: 3
+        prefix:
+          description: >-
+            The prefix of the IPv4 address.
+          type: integer
+          format: uint32
+          minimum: 1
+          maximum: 32
+          default: 32
+          x-field-uid: 4
+    Flow.RSVP.PathExplicitRouteType1FourByteASNumber:
+      description: >-
+        Class = EXPLICIT_ROUTE, Type1 ROUTE_RECORD C-Type = 1
+        Subobject: 4-byte AS number, C-Type: 32
+      type: object
+      properties:
+        l_bit:
+          x-field-pattern:
+            description: >-
+              The L bit is an attribute of the subobject.The L bit is set if the subobject represents a loose hop in the explicit route.
+              If the bit is not set, the subobject represents a strict hop in the explicit route.
+            format: integer
+            length: 1
+            default: 0
+            features: [count]
+          x-field-uid: 1
+        length:
+          description: >-
+            The Length contains the total length of the subobject in bytes,including L ,Type and Length fields.  
```

Comment:
space before comma should be removed and added after comma

Reply:
Done.